Cleanup main branch (#2)
astronomer/devrel-astro-airflow-workshop: README.md

Astronomer's Airflow + Astro workshop
=====================================

This repo contains workshops designed to give users an intro to Astro, Astronomer's flagship cloud product, as well as familiarize users with the latest features available in Airflow. 

This branch contains information for workshop hosts and contributors.

## Instructions and prerequisites for all workshops

> All workshops have their own branch in this repo (see #Choosing a workshop for more info about the different branched). The `README` in each branch contains instructions for the exercises. The file structure is the same as an Astro project, with an additional folder for `exercise-solutions` where relevant.

Reviewer: ```suggestion
All workshops have their own branch in this repo (see [Choosing a workshop](#choosing-a-workshop) for more info about the different branched). The `README` in each branch contains instructions for the exercises. The file structure is the same as an Astro project, with an additional folder for `exercise-solutions` where relevant.
```

Just testing if that works :)

> All workshops have their own branch in this repo (see [Choosing a workshop](#choosing-a-workshop) for more info about the different branched). The `README` in each branch contains instructions for the exercises. The file structure is the same as an Astro project, with an additional folder for `exercise-solutions` where relevant.

Reviewer: ```suggestion
All workshops have their own branch in this repo (see [Choosing a workshop](#choosing-a-workshop) for more info about the different branches). The `README` in each branch contains instructions for the exercises. The file structure is the same as an Astro project, with an additional folder for `exercise-solutions` where relevant.
```

Workshops are intended to be given in person or virtually, with one Astronomer presenter and at least one other Astronomer helper. Workshop content is not designed for participants to complete asynchronously. 

For each workshop, participants will need:

> - An Astro account (a free trial will work, but ensure it has access to the features covered in the workshop before beginning).

Reviewer: ```suggestion
- An Astro account (a [free trial](https://astronomer.io/try-astro) will work, but ensure it has access to the features covered in the workshop before beginning).
```

unless there is a reason you don't want to link :)

> - The Astro CLI

Reviewer: ```suggestion
- The [Astro CLI](https://docs.astronomer.io/astro/cli/install-cli)
```

> - Docker 4.0 or greater

Reviewer: ```suggestion
- [Docker](https://www.docker.com/) 4.0 or greater
```

## Choosing a workshop

> There are multiple variations of this workshop available on different branches in the repository:

Reviewer: ```suggestion
There are multiple workshops available on different branches in the repository:
```

Very nitty nit: Are they branches multiple variations of the same workshop or multiple workshops in the repo? 😅 I understood the rest of the document as being the latter.
